Sources/Private/CoreAnimation/CoreAnimationLayer.swift

```diff
@@ -94,8 +94,8 @@ final class CoreAnimationLayer: BaseAnimationLayer {
     }
   }
 
-  /// The parent `LottieAnimationView` that manages this layer
-  weak var animationView: LottieAnimationView?
+  /// The parent `LottieAnimationLayer` that manages this layer
+  weak var lottieAnimationLayer: LottieAnimationLayer?
```

Comment:
While I was looking in to this I noticed and fixed an unrelated issue:

We were passing in a reference to the parent `LottieAnimationView` so this layer could delegate some callbacks to its parent.

When using `LottieAnimationLayer` CALayer subclass on its own, there won't be a `LottieAnimationView`. We should pass in the `LottieAnimationLayer` instead, since that's where all of the functionality is implemented anyway.